app/controllers/books_controller.rb

```diff
@@ -5,5 +5,11 @@ def index
     @books = Book.all
     @available_books = @books.available
     @unavailable_books = @books.unavailable
+
+    if params[:search]
+      @books = Book.search(params[:search])
+    else
+      @books = Book.all
+    end
```

Comment:
Rather than doing the search here, you may want to consider breaking out a new resource/route called `book_searches` or something. But that's up to you. This is pretty simple so it might be totally fine as-is

Reply:
I agree with this!